<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo10_1，音乐播放器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lab1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>481</width>
      <height>481</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>53</y>
      <width>481</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{color: black; background-color: transparent; border:1px solid black;}</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>741</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>播放列表</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>添加</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/images/316.bmp</normaloff>:/icons/images/316.bmp</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="3">
      <widget class="QPushButton" name="btnRemove">
       <property name="text">
        <string>移除</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/images/318.bmp</normaloff>:/icons/images/318.bmp</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="5" colspan="3">
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>清空</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/images/214.bmp</normaloff>:/icons/images/214.bmp</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="8" colspan="2">
      <widget class="QPushButton" name="selectbtn">
       <property name="text">
        <string>选择图片</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>音乐播放列表</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>图片播放列表</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="5">
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item row="2" column="5" colspan="5">
      <widget class="QListView" name="listView"/>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btnPlay">
       <property name="toolTip">
        <string>播放</string>
       </property>
       <property name="text">
        <string>播放</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/images/620.bmp</normaloff>:/icons/images/620.bmp</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QPushButton" name="btnPause">
       <property name="toolTip">
        <string>暂停</string>
       </property>
       <property name="text">
        <string>暂停</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/images/622.bmp</normaloff>:/icons/images/622.bmp</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="btnStop">
       <property name="toolTip">
        <string>停止</string>
       </property>
       <property name="text">
        <string>停止</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/images/624.bmp</normaloff>:/icons/images/624.bmp</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="4" colspan="2">
      <widget class="QPushButton" name="btnPrevious">
       <property name="toolTip">
        <string>前一曲目</string>
       </property>
       <property name="text">
        <string>上一首</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/images/616.bmp</normaloff>:/icons/images/616.bmp</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QPushButton" name="btnNext">
       <property name="toolTip">
        <string>下一曲目</string>
       </property>
       <property name="text">
        <string>下一首</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/images/630.bmp</normaloff>:/icons/images/630.bmp</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>23</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="9">
      <widget class="QSlider" name="sliderVolumn">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="10">
      <widget class="Line" name="line">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="10">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="LabCurMedia">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>无曲目</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderPosition">
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="LabRatio">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>进度</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="8">
      <widget class="QPushButton" name="btnSound">
       <property name="text">
        <string>音量</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/images/volumn.bmp</normaloff>:/icons/images/volumn.bmp</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
